crates/recent_projects/src/recent_projects.rs

```diff
@@ -139,7 +139,7 @@ impl PickerDelegate for RecentProjectsDelegate {
     type ListItem = ListItem;
 
     fn placeholder_text(&self) -> Arc<str> {
-        "Recent Projects...".into()
+        "Search recent projects...".into()
```

Comment:
I changed this because it is more in line with the rest of the ui. "Search..." is used in the file finder, channel manage members overlay etc.